Library/ViewModels/CuratedProjectsViewModel.swift

```diff
@@ -6,11 +6,13 @@ import ReactiveSwift
 public protocol CuratedProjectsViewModelInputs {
   func configure(with categories: [KsApi.Category])
   func doneButtonTapped()
+  func projectTapped(_ project: Project?)
```

Comment:
The function that retrieves the value of `project` from the data source returns an optional value. Rather than unwrap the value in the VC, I thought it would be better to pass the value as an optional directly into the VM. This value _should_ never be nil.